### ISSUE TITLE
Form submits on enter key

### ISSUE THREAD

**Issue by nleo** [autocompletejs/autocomplete.js#10]
Put 'll', select 'Hello' with arrows, press enter, input text replaced with 'Hello', but before it form send with 'll' to server

```javascript
 if (keyCode == 13 && attr(result, "class").indexOf("open") != -1) {
    # Adding this dosn't help
    e.preventDefault();
    e.stopPropagation();
```

**Comment by nleo** [comment]
This helps

```javascript 

                        input.onkeydown = function(e) {
                            if (e.keyCode == 13 && attr(result, "class").indexOf("open") != -1) {
                                e.preventDefault();
                                e.stopPropagation();
                                return false;
                            }
                        }
```

**Comment by baptistedonaux** [comment]
Hi @nleo . It's true, It's really problematic. I work on the next version. I implement now this fix. Thank you for your feedback.

**Comment by baptistedonaux** [comment]
Implemented in d7feb8ae8516046d41b8a4507736372650f8e19e. Thank you for your feedback, code fix and encouragement.
